README.md

```diff
@@ -1,6 +1,20 @@
-# radar
+# TechLabs Semester Radar
+
+> The TechLabs Berlin semester radar is a web app that provides our Techies with an overview of all the activities & milestones in their TechLabs   semester. On the Radar, they can see all the information they need at a glance.
+
+## Resources
+
+- [TechLabs Semester Radar Page ](https://bln.techlabs.org/timeline) 
+- [Slack Channel](https://techlabs-community.slack.com/) 
+
+
+## About Radar
+> The Radar is built as a static single page app (SPA) with Nuxt.js using Vue.js as the frontend framework.
+  For detailed explanation on how things work, check out [Nuxt.js docs](https://nuxtjs.org).
+
+ ![Radar Project](assets/radar-image.png) 
```

Comment:
I'd appreciate it if you could scale down the screenshot down to something like 500px. A 600k file referenced in the README to show off the page seems a bit much 😄